docs-new/docs/contribute/general/contrib-guidelines-gen.md

```diff
@@ -43,3 +43,12 @@ please create an issue on the GitHub repository.
 * If you want to do the work on an issue,
   include that information in your description of the issue
   or in a comment to the issue.
+
+## Proposing New Work
```

Comment:
```suggestion
## Proposing new work
```
I think we agreed that we would follow the Google style guide here and use sentence case rather than title case for titles.